illumio/util/constants.py

```diff
@@ -46,8 +46,8 @@
 #: See https://www.iana.org/assignments/icmp-parameters/icmp-parameters.xhtml#icmp-parameters-types
 ICMP_TYPE_MAX = 255
 
-FQDN_REGEX = re.compile('(?=^.{4,253}$)(^((?!-)[a-zA-Z0-9-]{1,63}(?<!-)\.)+[a-zA-Z]{2,63}$)')
-HREF_REGEX = re.compile('^\/orgs\/\d+\/(?:sec_policy\/(?:active|draft)\/)?(?P<type>[a-zA-Z_]+)\/(?P<uid>[a-zA-Z0-9-]+)$')
+FQDN_REGEX = re.compile(r'(?=^.{4,253}$)(^((?!-)[a-zA-Z0-9-]{1,63}(?<!-)\.)+[a-zA-Z]{2,63}$)')
```

Comment:
What does this fix? These regexes were working before.

Reply:
All of the regexes had the following types of issues:

* PEP 8: W605 invalid escape sequence '\+'
* Redundant character escape '\+' in RegExp

Explanation:
In a regular expression, \w is a special sequence that matches any word character (alphanumeric characters and underscores). When using a regular string literal, the backslash \ is treated as an escape character, so \w is interpreted as an invalid escape sequence.

By using a raw string literal with r, the backslash is treated as a literal character, and \w is correctly interpreted as the special sequence for matching word characters.
So, the corrected regular expression pattern r'^\w+://' will match strings that start with one or more word characters followed by ://.  

For example:
http:// will match
ftp:// will match
file:// will match
:// will not match (because it doesn't start with word characters)

Using raw string literals is a good practice when working with regular expressions in Python to avoid issues with escape sequences.